### PR TITLE
Convert Bootstrap to Tachyons

### PR DESCRIPTION
**Pull request by nii236**

### CODE REVIEW

File: web/templates/session/form.html.haml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 - form_for @changeset, @action, fn f ->
   - if f.errors != [] do
-    .alert.alert-danger
+    .red
```
Comment (jgunnink) [inline web/templates/session/form.html.haml:3]: shouldn't it be `.alert-alert-danger.red`?

Are the alerts just the text? or is there a background? If background should be `.bg-red`

**Review by jgunnink — COMMENTED** [review]
Can you put screenshots of the different alerts up?

File: web/templates/session/form.html.haml (hunk 1 above)
Comment (nii236) [reply web/templates/session/form.html.haml:3]: `.alert` is a bootstrap thing

**Review by jgunnink — APPROVED** [review]
Looks good 👍

**Pull request merged**